dashboard/src/constants/env.ts

```diff
@@ -0,0 +1,50 @@
+type Environment = {
+  MOCKED: boolean;
+}
+
+const ENV: Environment = {
+  MOCKED: false,
+};
+
+// eslint-disable-next-line
+const covertToType = (value: any) => {
+  if (value === "true") {
+    return true;
+  }
+
+  if (value === "false") {
+    return false;
+  }
+
+  if (value === "null") {
+    return null;
+  }
+
+  if (value === "undefined") {
+    return undefined;
+  }
+
+  if (!isNaN(value)) {
+    return Number(value);
+  }
+
+  return value;
+};
+
+/* eslint-disable */
```

Comment:
eslint disabled?

Reply:
yep, there's a bunch of any that eslint reacts to